### ISSUE TITLE
Possible to make Threads?

### ISSUE THREAD

**Issue by michaelortnersanube** [Againstreality/Rocket.Chat.check_mk-integration#8]
Hello,

is it maybe possible to make threads for one service if it has more then one problem. As an example: If it has a problem, it should create the thread and then post from this service/server all new notification to this one thread.

**Comment by Againstreality** [comment]
good Idea but i havent worked with Rocket.chat or Check_mk for some years now.
